.circleci/config.yml

```diff
@@ -55,9 +55,28 @@ jobs:
       - checkout
       - setup_remote_docker:
           version: 20.10.6
+      - run:
+          name: Install Docker buildx
+          command: |
+            mkdir -p ~/.docker/cli-plugins
+            url="https://github.com/docker/buildx/releases/download/v0.8.0/buildx-v0.8.0.linux-amd64"
+            curl -sSL -o ~/.docker/cli-plugins/docker-buildx $url
+            chmod a+x ~/.docker/cli-plugins/docker-buildx
+      - run:
+          name: Enable Docker buildx
+          command: export DOCKER_CLI_EXPERIMENTAL=enabled
+      - run:
+          name: Initialize Docker buildx
+          command: |
+            docker buildx install
+            docker run --rm --privileged multiarch/qemu-user-static: --reset -p yes
+            docker rm -f buildx_buildkit_docker-multiarch0
+            docker buildx create --name docker-multiarch --platform linux/386,linux/amd64,linux/arm/v5,linux/arm/v6,linux/arm/v7,linux/arm64,linux/mips64le,linux/ppc64le,linux/riscv64,linux/s390x
```

Comment:
Isn't adding so many platforms going to make the image massive?

Reply:
It will create one image for each platform, so the image will keep his size. As we can see in Redis image for example  (https://hub.docker.com/_/redis?tab=tags), there is one image per platform